concepticondata/conceptlists/Tryon-1983-324.tsv

```diff
@@ -176,7 +176,7 @@ Tryon-1983-324-174	174	my	Possessives	2090	MY
 Tryon-1983-324-175	175	your sg.	Possessives	732	YOUR (SINGULAR)
 Tryon-1983-324-176	176	his	Possessives	2092	HIS OR HER
 Tryon-1983-324-177	177	our pl. incl.	Possessives	780	OUR
-Tryon-1983-324-178	178	our pl. Excl.	Possessives	780	OUR
+Tryon-1983-324-178	178	our pl. excl.	Possessives	780	OUR
```

Comment:
Ups... I guess we should be careful and already now add another concept set: 

* OUR
* OUR (INCLUSIVE)
* OUR (EXCLUSIVE)

It's annoying, but it would be more consistent.